library/src/skins/default/src/sass/modules/tool/sitemanage/_sitemanage.scss

```diff
@@ -426,7 +426,7 @@
   }
   .removeTool {
     text-decoration: none !important;
-    color: var(--errorBanner-color);
+    color: var(--errorBanner-color) !important;
```

Comment:
Why do you need the !important? What is overriding it?

Reply:
<a> is overriding

Reply:
What scss file is that coming from? Is there no way the order can be corrected?